framework/src/controller/application.js

```diff
@@ -84,6 +84,10 @@ module.exports = class Application {
 			};
 		}
 
+		if (typeof label === 'function') {
```

Comment:
if the `label` is a function, above `${label} doesn't make sense.
Also, prefer to be consistent in type

Reply:
You didn't noticed the line after that condition. Label can be a string or a function that generate this string. 

That's a common pattern used along many other NodeJS project including PgPromise.